src/ImageProcessor.Web/Caching/IImageCache.cs

```diff
@@ -41,6 +41,16 @@ public interface IImageCache
         int BrowserMaxDays { get; set; }
 
         /// <summary>
+        /// Gets or sets the maximum number folder levels to nest the cached images.
+        /// </summary>
+        int FolderDepth { get; set; }
```

Comment:
Good point, people might be implementing this for blob storage purposes. Alternatively you could make this an extension method for now so it's not a breaking change (but then you also can't enforce implementing it). 
However, I can't find any examples of anybody actually implementing this interface, though it's possible it's in use privately.